web/iframe.html

```diff
@@ -83,9 +83,10 @@
     function receiveMessages(event) {
       try {
         const data = typeof event.data === 'string' ? JSON.parse(event.data) : event.data;
+        const isItHash = data.hash && data.hash.indexOf("#") === 0;
```

Comment:
Why do you need this check?

Reply:
postMessage can be called by everyone on the parent window. If someone decides to send some data with post message and instead of hash it puts some random string, it will cause redirection to the random URL with the string (because of the replace method). But If this string starts with "#", only hash part will be change, url will stay the same and we protect our self from dangerous redirections.  @igoradamenko

Reply:
I'm not sure that it's a big deal, but thanks for the detailed explanation. I didn't think about it :-)